kymatio/scattering3d/backend/backend_skcuda.py

```diff
@@ -1,19 +1,25 @@
-# The `skcuda` backend for Scattering3D is currently under development.
-# See issue #179.
+import torch
+from skcuda import cublas
 
 NAME = 'skcuda'
 
 
-skcuda_notimplementederror =\
-"""The \"skcuda\" backend for Scattering3D is currently under development.
-To perform 3D scattering, please set KYMATIO_BACKEND_3D=\"torch\"."""
+def iscomplex(input):
+    return input.size(-1) == 2
+
 
 def to_complex(input):
```

Comment:
needs a unit test

Reply:
it's already so the the torch backend, let's make a specific PR for unit tests then, later on

Reply:
yeah. that way we don't fall behind with test coverage.